### DISCUSSION TITLE
Album thumbnails

### DISCUSSION THREAD

**Discussion by firstchanceongit** [rern/rAudio#231] [General]
Not sure it's a bug (or a feature ;-))

My music collection mirrors the "physical facts", i.e. each CD gets its own folder and an album box gets as many subfolders as the number of CDs it consists of. The files are properly tagged, i.e. 01, 02 and 03 etc. for each CD in a subfolder, track numbers starting from 01 within each CD/subfolder. The album name is the same for all CDs contained in a multi-CD album (e.g. "White Album" from the Beatles) but is different if it the box is a collection of albums (e.g. all the studio albums contained in the Beatles Stereo Box where each album gets its own album name and just the folder structure reflects the fact that it has been physically distributed as a box set).

This typically results in one thumbnail in the library album art view for each CD/subfolder (just one single album thumbnail for all CDs/subfolders with the same album name even though split over several sub-folcers seems sometimes preferrable (i.e. just one thumbnail for the "White Album", but I appreaciate that this might not be the best solution in all instances - so, at least I think the current approach seems justifyable).

What I do not understand, however, is why I get about 20 album art thumbnails for a compilation album, i.e. a soundtrack with music from 20 different bands/interprets/composers even though the album name tag is the same for all tracks and all files are stored in the same folder. This massively inflates the album art view. Apparently, not just the album name or folder structure is considered when creating the album art view/thumbnails but also some more information from the file tags. Can that be changed?

**Comment by rern** [comment]
>...
What I do not understand, however, is why I get about 20 album art thumbnails for a compilation album, i.e. a soundtrack with music from 20 different bands/interprets/composers even though the album name tag is the same for all tracks and all files are stored in the same folder. This massively inflates the album art view. Apparently, not just the album name or folder structure is considered when creating the album art view/thumbnails but also some more information from the file tags. Can that be changed?

Tracks with the same `album title` AND `album artist` (fall back to `artist` if not available) are in the same album.
Your compilation album has the same `album title` with no `album artist` which falls back to different `artist` tags.
Set `album artist` tag to something like "Various artists".

**Reply by firstchanceongit** [reply]
Thank you for the clarification. I was in fact wondering about the logic behind.

**Reply by firstchanceongit** [reply]
I checked the respective album: "album artist" was in fact already tagged with "VA". ("artist" varies from file to file, "album title" is identical for all files.) So, there must be some other reason ... :-/

EDIT: I have to correct myself - you were right. I usually use Tag&Rename to tag my files and it correctly showed "VA" as "album artist" as stated above. When I double checked with MP3tag and it showed the the "album artist" field to be empty. Retagging with MP3tag solved the issue. Very strange ...

**Reply by rern** [reply]
Post the duplicate albums listed in `/srv/http/data/mpd/album`.
